### PR TITLE
Mapping guide

### PR DESCRIPTION
**Pull request by pipliggins**
Created a mapping guide for clinical data parsers.

Still a work in progress - but input on whether I've got the existing things right, and what else would be useful to go in to make sure everyone's on the same page and we're consistent with how things are being applied would be great.

Addresses #80

### CODE REVIEW

File: docs/guide.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+```
+[subject.date_death]
+  field = "dsstdtc"
+  if = { dsterm = 4, description = "Outcome date" }
+
+[visit.outcome]
+  field = "dsterm"
+  description = "Outcome date"
+  values = {
```
Comment (abhidg) [inline docs/guide.md:52]: should be `[visit.outcome.values]`

File: docs/guide.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    3 = "transferred"
+    4 = "death"
+    5 = "discharged"
+  }
+```
+
+## Phase descriptors
+Phase descriptors are used within the 'visit' and 'observations' tables to describe which time period within the study the data refers to. One of three descriptors can be used:
+
+**admission**: Refers to data collected upon patient admission; general patient data (age, gender etc) plus any co-morbidities. Any symptoms/observations recorded at admission may refer to the day of admission, or a time-period leading up to the day of hospital admission.
```
Comment (abhidg) [inline docs/guide.md:64]: I think we decided to remove phase from visit? As nearly all visit data is in the `study` phase. Patient demographics /co-morbidities are in subject.

File: docs/guide.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+**study**: Data collected during a single hospital visit. Can refer to a single event within the visit period, or something present throughout the time spent in hospital.
+
+**follow-up**: Data collected during follow-up visits. As with the 'admission' phase, the attribute can refer to the single day of the follow-up survey being completed, or to a time period between hospital discharge and the date of the follow-up form being completed.
```
Comment (abhidg) [inline docs/guide.md:68]: -> `followup`

**Review by abhidg — COMMENTED** [review]
Thanks!

**Comment by ekamau** [comment]
Nicely done. Will look at this more thoroughly tomorrow morning.

File: docs/guide.md (hunk 2 above)
Comment (pipliggins) [reply docs/guide.md:64]: 'phase' is still present in the visit schema file - if the plan is to remove it I can take this out. Noted about the co-morbidities, I'll take that out.

File: docs/guide.md [hunk 4]
```diff
@@ -49,22 +49,23 @@ However, below is a valid example of `if` statement usage; `dsstdtc` will usuall
...
 
-**follow-up**: Data collected during follow-up visits. As with the 'admission' phase, the attribute can refer to the single day of the follow-up survey being completed, or to a time period between hospital discharge and the date of the follow-up form being completed.
+**followup**: Data collected during follow-up visits. As with the 'admission' phase, the attribute can refer to the single day of the follow-up survey being completed, or to a time period between hospital discharge and the date of the follow-up form being completed.
 
 In addition to the three phase descriptors, within the observations table the `duration_type` attribute can be used to distinguish between an 'event', which denotes an observation that occurs once or multiple times within the observation period, or a 'block' to refers to something present within the entire observation period. In this case, the observation period is given by the interval between `start_date` and `date`, providing a more specific date range on top of the phase descriptors.
```
Comment (ekamau) [inline docs/guide.md:75]: We probably discussed this: what are text examples of 'duration_type'? how would I write it in the parser?

**Review by ekamau — COMMENTED** [review]
Probaby add examples or text entries for 'duration_type' - how it would appear or be entered in the parser. I haven't still worked it out  ...

File: docs/guide.md (hunk 4 above)
Comment (pipliggins) [reply docs/guide.md:75]: Do you mean something different to the example below?

**Comment by abhidg** [comment]
Closing as related branch mapping-guide-edits has been merged into main

**Pull request closed without merging**